### PR TITLE
Update core schema to current spec

### LINKED ISSUE #141: Update urn schema URIs
The [used URIs](https://github.com/osiam/scim-schema/blob/master/src/main/java/org/osiam/resources/scim/Constants.java#L31) in this repo are not up to date with [the specification defined URIs](https://tools.ietf.org/html/draft-ietf-scim-core-schema-22#section-10.4). This is just one task of the obvious todo of the adjustment to the scim specfication.

### PR DESCRIPTION
**Pull request by sputnik27**
Fixes #141

### CODE REVIEW

**Comment by sputnik27** [comment]
Relevant: This is related to #141, I did miss that issue.

**Comment by tkrille** [comment]
please update the changelog, too.

**Comment by tkrille** [comment]
Please remove the commit that deals with the `@Serializable` stuff.

**Comment by tkrille** [comment]
Changelog is missing again.

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 - Remove unused meta classes and package
 - Deprecate User/Group/Resource#setSchemas(...)
+- Update Schema URIs to SCIM Draft v. 19
```
Comment (tkrille) [inline CHANGELOG.md]: Typo: `schema`

Comment (tkrille) [inline CHANGELOG.md]: Typo: `draft`

Comment (tkrille) [inline CHANGELOG.md]: I think `v.` is necessary. The official label is `draft-ietf-scim-api-19`.

Comment (tkrille) [inline CHANGELOG.md]: > Schema URIs

I think they're actually URNs whicht should be explicitly stated here.

**Pull request merged**